meta-resin-common/recipes-core/initrdscripts/files/rootfs

```diff
@@ -31,6 +31,11 @@ rootfs_run() {
                 bootparam_root="/dev/disk/by-partuuid/$root_uuid"
             fi
 
+            if [ "`echo ${bootparam_root} | cut -c1-6`" = "LABEL=" ]; then
+                root_uuid=`echo $bootparam_root | cut -c7-`
```

Comment:
I tend to use $() in preference to backticks but I don't know if it makes much difference.